config/deployment.yml

```diff
@@ -43,19 +43,25 @@ spec:
         volumeMounts:
         - name: template-fs
           mountPath: /etc/kappctrl-mem-tmp
-        securityContext:
-          runAsUser: 1000
-          runAsGroup: 2000
+        - name: home
+          mountPath: /home/kapp-controller
         ports:
         - containerPort: #@ data.values.api_port
           name: api
           protocol: TCP
-      securityContext:
-        fsGroup: 3000
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: false #! TODO(bmo): don't write to the root ca-bundle
```

Comment:
Ideally this could be `true` except for the `/etc/pki/tls/certs/ca-bundle.crt` file I mentioned we're writing to.